examples/example-column-hidden.html

```diff
@@ -0,0 +1,378 @@
+      text-align: right;
+      font-size: 10px;
+    }
+
+    .slick-row.selected .cell-selection {
+      background-color: transparent; /* show default selected row background */
+    }
+  </style>
+</head>
+<body>
+<div style="position:relative">
+  <div style="width:600px;">
+    <div class="grid-header" style="width:100%">
+      <label>SlickGrid</label>
+      <span style="float:right;margin:0px 4px auto 6px;background:transparent;cursor:pointer;" class="slick-icon-search" title="Toggle search panel"
+            onclick="toggleFilterRow()"></span>
+    </div>
+    <div id="myGrid" style="width:100%;height:500px;"></div>
+    <div id="pager" style="width:100%;height:20px;"></div>
+  </div>
+
+  <div class="options-panel">
+    <b>Search:</b>
+    <hr/>
+    <div style="padding:6px;">
+      <label style="width:200px;float:left">Show tasks with % at least: </label>
+
+      <div style="padding:2px;">
+        <input style="width:100px;display:inline-block;" id="pcSlider" type="range" min="1" max="100" value="1">
+      </div>
+      <br/>
+      <label style="width:200px;float:left">And title including:</label>
+      <input type=text id="txtSearch" style="width:100px;">
+      <br/><br/>
+      <button id="btnSelectRows">Select first 10 rows</button>
+      <br/><br/>
+      <input id="chkHideColumn" type="checkbox"/> Hide Duration Column
+      <br/>
+
+      <h2>Demonstrates:</h2>
+      <ul>
+        <li>a filtered Model (DataView) as a data source instead of a simple array</li>
+        <li>grid reacting to model events (onRowCountChanged, onRowsChanged)</li>
+        <li>
+          <b>FAST</b> DataView recalculation and <b>real-time</b> grid updating in response to data changes.<br/>
+          The grid holds <b>50'000</b> rows, yet you are able to sort, filter, scroll, navigate and edit as if it had 50
+          rows.
+        </li>
+        <li>adding new rows, bidirectional sorting</li>
+        <li>column options: cannotTriggerInsert</li>
+        <li>events: onCellChange, onAddNewRow, onKeyDown, onSelectedRowsChanged, onSort</li>
+        <li><font color=red>NOTE:</font> all filters are immediately applied to new/edited rows</li>
+        <li>Handling row selection against model changes.</li>
+        <li>Paging.</li>
+        <li>inline filter panel</li>
+      </ul>
+        <h2>View Source:</h2>
+        <ul>
+            <li><A href="https://github.com/6pac/SlickGrid/blob/master/examples/example-column-visible.html" target="_sourcewindow"> View the source for this example on Github</a></li>
+        </ul>
+
+    </div>
+  </div>
+</div>
+
+<div id="inlineFilterPanel" style="display:none;background:#dddddd;color:black;">
+  Show tasks with title including <input type="text" id="txtSearch2">
+  and % at least &nbsp;
+  <input style="width:100px;display:inline-block;" id="pcSlider2" type="range" min="1" max="100" value="1">
+</div>
+
+<script src="../lib/firebugx.js"></script>
+
+<script src="../lib/jquery-3.1.0.js"></script>
```

Comment:
we'll eventually have to migrate this example also to be without jQuery, I started converting a few but will probably do the rest after work today